doc/README.md

```diff
@@ -89,6 +89,16 @@ Alternatively, you can use some specific targets:
 Also note the `-with-doc yes` option of `./configure` to enable the
 build of the documentation as part of the default make target.
 
+If you're editing Sphinx documentation, set SPHINXWARNERROR to 0
+to avoid treating Sphinx warnings as errors.  Otherwise, Sphinx quits
+upon detecting the first warning.  You can set this on the Sphinx `make`
+command line or as an environment variable: `make sphinx SPINXWARNERROR=0` or
+
+```
+export SPHINXWARNERROR=0
+   :
```

Comment:
Not sure what this `:` is supposed to mean, and note that using code blocks doesn't prevent you from using bullet points like you did before.

Reply:
It's a poor man's vertical ellipsis.  You've not seen this before?  The idea being that you could put the `export` in a script or even in your .bashrc.  The genuine item is `⋮` but a colon is a good approximation.

I'm not set up to view the formatted .md file (hence my off topic comment), so I didn't have a way to check the appearance.  Maybe how to do that is obvious to most people, but it's another detail that would be good in the setup directions.

So I can do it like this?

```
- `make sphinx SPINXWARNERROR=0`

-
<3 backquotes>
export SPHINXWARNERROR=0
make sphinx
<3 backquotes>
```

Reply:
Or ~~~ instead of ```

Reply:
Your best shot is using GitHub itself as a viewer: you always have a button to display rich diffs in PRs (rendering the Markdown) and you also have an in-editor browser with preview.

FWIW:

```
- `make sphinx SPINXWARNERROR=0`

- ~~~
  export SPHINXWARNERROR=0
    ⋮
  make sphinx
  ~~~
```

is rendered like this:

- `make sphinx SPINXWARNERROR=0`

- ~~~
  export SPHINXWARNERROR=0
    ⋮
  make sphinx
  ~~~

but what you suggested doesn't work: it gives an empty bullet.

Reply:
> in-editor browser with preview

Not in my editor :-(.

Fixed this, go ahead and merge anytime.